card-style.cls

```diff
@@ -12,6 +12,7 @@
 \RequirePackage[warnings-off={mathtools-colon,mathtools-overbracket}]{unicode-math}
 \RequirePackage{multicol}
 \RequirePackage{xparse}
+\RequirePackage[left=4mm, right=4mm, top=4mm, bottom=4mm]{geometry}
```

Comment:
we also have half of a 4mm colored border on the page, for 2mm colored + 2mm white margin. A bit more colored and less white in the corners because the colored border has rounded corners.